types/recharts/index.d.ts

```diff
@@ -645,13 +646,13 @@ export interface TooltipProps {
 	itemStyle?: any;
 	wrapperStyle?: any;
 	labelStyle?: any;
-	cursor?: boolean | any | React.ReactElement<any>;
+	cursor?: boolean | any | React.ReactElement<any> | React.StatelessComponent<any>;
```

Comment:
Is React.StatelessComponent also a ReactElement? since the docs are saying of type `Boolean | Object | ReactElement`

Reply:
Yes, React.StatelessComponent is a React.ReactElement, but without class definition. See https://facebook.github.io/react/docs/components-and-props.html#extracting-components